src/main/java/uk/gov/hmcts/darts/common/repository/HearingRepository.java

```diff
@@ -44,5 +44,11 @@ AND upper(cr.name) = upper(:courtroom)
     )
     Optional<HearingEntity> findHearing(String courthouse, String courtroom, String caseNumber, LocalDate date);
 
+    @Query("""
```

Comment:
Would an `EXISTS` query be possible and more appropriate for performance reasons? As the system ages, the number of hearings associated with a courthouse will become significant, and `COUNT` may involve a full table scan, wheres `EXISTS` will stop after finding the first match.